src/util/env.js

```diff
@@ -11,11 +11,13 @@ export const inBrowser =
 // detect devtools
 export const devtools = inBrowser && window.__VUE_DEVTOOLS_GLOBAL_HOOK__
 
-// UA sniffing for working around browser-specific quirks
+// UA sniffing for working around browser-specific quirks, distinguish UIWebView from WKWebView by indexedDB
 const UA = inBrowser && window.navigator.userAgent.toLowerCase()
 export const isIE9 = UA && UA.indexOf('msie 9.0') > 0
 export const isAndroid = UA && UA.indexOf('android') > 0
 export const isIos = UA && /(iphone|ipad|ipod|ios)/i.test(UA)
+export const isIos93 = isIos && UA.indexOf('os 9_3') > 0
+export const isIndexedDB = window.indexedDB || window.webkitIndexedDB || window.mozIndexedDB || false
```

Comment:
In addition: since this check is only used for the specific case of iOS (detecting WKWebView), there is no need to check `webkitIndexedDB` and `mozIndexedDB`.